composer/datasets/in_context_learning_evaluation.py

```diff
@@ -125,6 +128,17 @@ class InContextLearningQATaskDataset(Dataset):
         fewshot_random_seed (int): Random seed to use for fewshot sampling
     """
 
+    @classmethod
+    def read_dataset(cls, dataset):
```

Comment:
Is there a reason this needs to be a class method instead of a generic function? Also use a for loop instead of a lambda inside a map

Reply:
I can make it a regular function